pkg/kudoctl/cmd/plan.go

```diff
@@ -46,18 +48,21 @@ func NewPlanHistoryCmd() *cobra.Command {
 }
 
 //NewPlanStatusCmd creates a new command that shows the status of an instance by looking at its current plan
-func NewPlanStatusCmd() *cobra.Command {
-	options := plan.DefaultStatusOptions
+func NewPlanStatusCmd(out io.Writer) *cobra.Command {
+	options := &plan.Options{Out: out}
 	statusCmd := &cobra.Command{
 		Use:     "status",
 		Short:   "Shows the status of all plans to an particular instance.",
 		Example: planStatuExample,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return plan.RunStatus(cmd, options, &Settings)
+			return plan.Status(options, &Settings)
 		},
 	}
 
 	statusCmd.Flags().StringVar(&options.Instance, "instance", "", "The instance name available from 'kubectl get instances'")
+	if err := statusCmd.MarkFlagRequired("instance"); err != nil {
+		panic(err)
```

Comment:
we don't panic much. wdyt about:
```suggestion
		clog.V(0).Print("failed to makr --instance flag as required: %v", err)
		os.Exit(1) 
```